scripts/dependency-graph.js

```diff
@@ -1,189 +1,153 @@
-const Arborist = require('@npmcli/arborist')
-const { readFileSync } = require('fs')
-const { join } = require('path')
-const log = require('proc-log')
-const { run, CWD, pkg, fs } = require('./util.js')
+const Arborist = require("@npmcli/arborist");
+const { readFileSync, writeFile } = require("fs");
+const { join } = require("path");
 
 // Generates our dependency graph documents in DEPENDENCIES.md.
 
 // To re-create npm-cli-repos.txt run:
 // npx -p @npmcli/stafftools gh repos --json | json -a name | sort > scripts/npm-cli-repos.txt
-const repos = readFileSync(join(CWD, 'scripts', 'npm-cli-repos.txt'), 'utf-8').trim().split('\n')
+const repos = readFileSync(join(__dirname, "npm-cli-repos.txt"), "utf-8").trim().split("\n");
 
 // these have a different package name than the repo name, and are ours.
 const aliases = {
-  semver: 'node-semver',
-  abbrev: 'abbrev-js',
-}
+  semver: "node-semver",
+  abbrev: "abbrev-js",
+};
 
 // These are entries in npm-cli-repos.txt that correlate to namespaced repos.
 // If we see a bare package with just this name, it's NOT ours
 const namespaced = [
-  'arborist',
-  'config',
-  'disparity-colors',
-  'eslint-config',
-  'exec',
-  'fs',
-  'git',
-  'installed-package-contents',
-  'lint',
-  'map-workspaces',
-  'metavuln-calculator',
-  'move-file',
-  'name-from-folder',
-  'node-gyp',
-  'package-json',
-  'promise-spawn',
-  'run-script',
-  'template-oss',
-]
-
-function isOurs (name) {
-  if (name.startsWith('libnpm')) {
-    return true
-  }
-  if (name.startsWith('@npmcli')) {
-    return true
-  }
-  if (aliases[name]) {
-    return true
+  "arborist",
+  "config",
+  "disparity-colors",
+  "eslint-config",
+  "exec",
+  "fs",
+  "git",
+  "installed-package-contents",
+  "lint",
+  "map-workspaces",
+  "metavuln-calculator",
+  "move-file",
+  "name-from-folder",
+  "node-gyp",
+  "package-json",
+  "promise-spawn",
+  "run-script",
+  "template-oss",
+];
+
+function isOurs(name) {
+  if (name.startsWith("libnpm") || name.startsWith("@npmcli") || aliases[name]) {
+    return true;
   }
-  // this will prevent e.g. `fs` from being mistaken as ours
   if (namespaced.includes(name)) {
-    return false
+    return false;
   }
-  if (repos.includes(name)) {
-    return true
-  }
-  return false
+  return repos.includes(name);
 }
 
-function escapeName (name) {
-  if (name.startsWith('@')) {
-    return `${stripName(name)}["${name}"]`
-  }
-  return name
+function escapeName(name) {
+  return name.startsWith("@") ? `${stripName(name)}["${name}"]` : name;
```

Comment:
We avoid ternary if statements if at all possible.  They are extremely hard to read especially as they get nested.